src/operator/nn/mkldnn/mkldnn_convolution.cc

```diff
@@ -126,13 +127,12 @@ mkldnn::convolution_forward::primitive_desc GetConvFwdImpl(const MKLDNNConvFullP
 
   if (param.conv_param.dilate.ndim() == 0 && bias_md_ptr == nullptr) {
     mkldnn::convolution_forward::desc desc(prop, mkldnn::algorithm::convolution_direct, data_md,
-                                           weight_md, out_md, strides, padding, padding,
-                                           mkldnn::padding_kind::zero);
```

Comment:
Is `padding_kind`  disabled or `zero` becomes default option from MKL-DNN v1.0?

Reply:
mkldnn_padding_kind_t  Not used anymore
https://intel.github.io/mkl-dnn/dev_guide_transition_to_v1.html